dags/utils/slack_operator.py

```diff
@@ -1,11 +1,58 @@
 from airflow.hooks.base_hook import BaseHook
 from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
 
+from cron_descriptor import get_description
```

Comment:
i don't know how much this matters, but it might not hurt to move this into the `format_schedule` function, similar to what we've done with the pendulum import 😇

Reply:
Great call - thanks for catching this little detail that would add up as this gets executed however many zillion times. https://github.com/cityofaustin/atd-airflow/pull/264/commits/6ee86f7aa8c333f86278caf005068b33baf36d3f